player/video.c

```diff
@@ -106,6 +106,7 @@ void reset_av_state(struct MPContext *mpctx)
     mpctx->logged_async_diff = -1;
     mpctx->num_past_frames = 0;
     mpctx->total_avsync_change = 0;
+    mpctx->speed_changed = true;
```

Comment:
Feels dirty to have `speed_changed = true` in a function called `reset_av_state`. Maybe we could move this out of the function instead

Reply:
Yeah this was supposed to be in player/command.c actually.